scylla/tests/conftest.py

```diff
@@ -30,7 +30,11 @@ def db_instance():
 
 @pytest.fixture()
 def mock_db_data():
-    f_name = os.path.join(os.path.dirname(__file__), 'fixtures', 'scylla_metrics.txt')
+    if os.environ['SCYLLA_VERSION'] == '5.2.6':
```

Comment:
Maybe to be a bit more future proof:

```suggestion
    if os.environ['SCYLLA_VERSION'].startswith('5.'):
```